onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -162,6 +162,9 @@ ORT_STATUS_PTR CreateTensorImpl(MLDataType ml_type, const int64_t* shape, size_t
     shapes[i] = shape[i];
   }
 
+  if ( std::any_of(shapes.begin(), shapes.end(), [](int i){return i<0;}) )
```

Comment:
nit: could do this check when populating the 'shapes' vector. would also be good to run clang-format so the formatting is consistent.

The build failure is due to shapes containing int64_t but your lambda has an 'int' for the argument.

Reply:
Reformatted the code and moved the check inside the for loop. clang-format also corrected formatting in another part of the file as well.